fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java

```diff
@@ -253,6 +253,7 @@ public class LoanAccountData {
     private CollectionData delinquent;
     private DelinquencyRangeData delinquencyRange;
     private LocalDate lastClosedBusinessDate;
+    private Boolean chargedOff;
```

Comment:
Since you touched the LoanAccountData, do you mind extending the LoanAccountDataV1 Avro schema too? (Same for LoanApplicationTimelineData, LoanSummaryData)

Reply:
True!